helm/templates/deployer/deployerServiceAccount.yaml

```
@@ -0,0 +1,11 @@
+{{ if eq .Release.Namespace .Values.deployer.namespace }}
```

Couldn't we grab this directly from `.Release.Namespace`? I was suggesting something like
```suggestion
{{ if .Values.deployerServiceAccount.enabled }}
```